R/functions-eager.R

```diff
@@ -100,39 +100,41 @@ pl__concat <- function(
     )
   }
 
-  if (how == "align") {
+  if (startsWith(how, "align")) {
     if (!is_polars_df(first) && !is_polars_lf(first)) {
-      abort(r"("align" strategy is only supported on DataFrames and LazyFrames.)")
+      abort(paste0("\"", how, "\" strategy is only supported on DataFrames and LazyFrames.)"))
```

Comment:
```suggestion
      abort(sprintf("'%s' strategy is only supported on DataFrames and LazyFrames.", how))
```

...And please add snapshot tests for this error.